src/lavague/__init__.py

```diff
@@ -0,0 +1,20 @@
+import os
+import warnings
+
+def telemetry_warning():
+    telemetry_var = os.getenv("TELEMETRY_VAR") 
+    if telemetry_var != "NONE":
+        warning_message = (
+            "\033[93mTelemetry is turned on. To turn off telemetry, set your TELEMETRY_VAR to 'NONE'\033[0m"
+        )
+        warnings.warn(warning_message, UserWarning)
+
+def exec_warning():
+    warning_message = (
+        "\033[91mSecurity warning: This package executed LLM-generated code with the exec() method. "
+        "Please consider running this package in a sandboxed environment\033[0m"
```

Comment:
I think this is a bit too scary, most people won't set up a sandbox. Maybe better would be: Only use it with models and websites you trust.